package-system/SPIRVCross/build_config.json

```diff
@@ -1,29 +1,16 @@
 {
-   "git_url":"https://github.com/o3de/SPIRV-Cross.git",
-   "git_tag":"release-2021.04.29",
+   "git_url":"https://github.com/KhronosGroup/SPIRV-Cross.git",
+   "git_tag":"vulkan-sdk-1.3.275.0",
    "package_name":"SPIRVCross",
-   "package_version":"2021.04.29-rev1",
-   "package_url":"https://github.com/03de/SPIRV-Cross",
+   "package_version":"1.3.275.0-rev1",
+   "package_url":"https://github.com/KhronosGroup/SPIRV-Cross",
    "package_license":"Apache-2.0",
    "package_license_file":"LICENSE",
    "cmake_find_target":"FindSPIRVCross.cmake",
    "Platforms":{
       "Windows":{
          "Windows":{
             "cmake_find_template":"FindSPIRVCross.cmake.Windows",
-            "cmake_generate_args_debug":[
-               "-G",
-               "\"Visual Studio 16 2019\"",
-               "-A",
-               "x64",
-               "-T",
-               "host=x64",
-               "-DSPIRV_CROSS_CLI=ON",
-               "-DSPIRV_CROSS_SHARED=OFF",
-               "-DCMAKE_INSTALL_LIBDIR=\"lib/Debug\"",
-               "-DCMAKE_INSTALL_BINDIR=\"bin/Debug\"",
-               "-DCMAKE_BUILD_TYPE=Debug"
-            ],
             "cmake_generate_args_release":[
```

Comment:
Minor recommendation to remove the Generator for Windows
```
               "-G",
               "\"Visual Studio 16 2019\"", 
```
CMake will use the most recent version of Visual Studio on the build machine which is what we did on our Jenkins machines so we don't have to update these files every time we update the build nodes with a new version of Visual Studio.

Note that for platforms like Linux you do want to keep the `G Ninja` because it might default to GCC, and I think you need to leave `Xcode` for Mac too.

Reply:
Yep. I had to make this change as the node didn't have VS2019